src/main/webapp/i18n/de/exam.json

```diff
@@ -132,6 +132,13 @@
             "finishExam": "Beenden",
             "progress": "Aufgabe {{ current }} von {{ all }}",
             "progressSmall": "{{ current }}/{{ all }}",
+            "exerciseType": "Aufgaben Typ",
+            "examOverview": "Klausur Übersicht",
+            "examStatus": "Aufgaben Status",
+            "examExercises": "Klausur Aufgaben:",
+            "exercisePoints": "Punkte",
+            "exerciseName": "Aufgaben Name:",
+            "noStudentExam": "Du bist nicht für die Klausur {{ examTitle }} angemeldet. Bitte wende dich an deinen Dozenten.",
```

Comment:
Is `noStudentExam` included twice? might be due to a resolved merge conflict. Please avoid "Dozent" 😄

Reply:
Yes that was just a merge conflict as can be seen be the comma. Thanks for noticing this.